assets/css/styles.scss

```diff
@@ -2,21 +2,39 @@ body {
   font-family: "Open Sans", Helvetica, Arial, sans-serif;
 }
 
-footer {
-  padding: 30px 0;
-  margin-top: 30px;
-  border-top: 1px solid $gray;
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Raleway", Helvetica, Arial, sans-serif;
+}
 
+.h2 {
   @media screen and (max-width: $screen-md-min) {
-    display: none;
+    margin-bottom: 10px;
   }
 }
 
-.navbar-brand {
-  font-family: "Raleway", Helvetica, Arial, sans-serif;
+.main-container {
+  min-height: 400px;
 }
 
-h1, h2, h3, h4, h5, h6 {
+
+.footer {
+  width: 100%;
+  padding: $base-margin 0;
+  margin-top: $base-margin-l;
+  border-top: 1px solid $dark;
+  background-color: $main-blue;
```

Comment:
That blue footer is weird 😜

Reply:
You don't like it :( - at the end it will be red + white color as on the website.